mteb/tasks/Classification/swa/SwahiliNewsClassification.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+# from ....abstasks import AbsTaskClassification
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SwahiliNewsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SwahiliNewsClassification",
+        description="Dataset for Swahili News Classification, categorized with 5 domains. Building and Optimizing Swahili Language Models: Techniques, Embeddings, and Datasets",
```

Comment:
which 5 domains?

Reply:
Reason for Adding the Dataset and domain of the classification:

Swahili is spoken by 100-150 million people across East Africa. In Tanzania, for example, it is one of the primary national languages and the official language of instruction in all schools. News in Swahili is an integral part of Tanzania's media sphere.
News contributes to education, technology, and a country's economic growth, and news in local languages plays an important cultural role in many African countries. In the modern age, however, African languages in news and other spheres are at risk of being lost as English becomes the dominant language in online spaces.
The Swahili news dataset was created to bridge the gap in utilizing the Swahili language to create NLP technologies. It aims to assist AI practitioners in Tanzania and across Africa in honing their NLP skills to address various challenges within organizations or societies related to the Swahili language. The Swahili news dataset, sourced from multiple websites providing news in Swahili, is a valuable resource for NLP research and development.
The dataset was curated explicitly for text classification tasks, categorizing news content into six distinct topics. This categorization facilitates the development of robust NLP models that can more effectively understand and process Swahili text. 
Six Domains:
Local News (Kitaifa):
News concerning local events, politics, and developments within Tanzania.
International News (Kimataifa):
Coverage of global events and news stories affecting the international community.
Finance News (Uchumi):
Financial and economic news, including market trends, monetary policies, and business updates.
Health News (Afya):
Information and updates on health-related topics, medical research, public health issues, and wellness.
Sports News (Michezo):
News related to sports events, athletes, competitions, and sports culture locally and internationally.
Entertainment News (Burudani):
Coverage of entertainment industry news, including celebrity updates, music, movies, and cultural events.
By integrating this dataset into MTEB, we aim to support the development of NLP models capable of understanding and processing Swahili text across these vital domains, thus promoting linguistic diversity and technological advancement in East Africa. The high accuracy of this dataset is attributed to the human annotation process involved in its creation.